modal_proto/api.proto

```diff
@@ -113,7 +113,6 @@ message AppDeployResponse {
 message AppGetByDeploymentNameRequest {
   DeploymentNamespace namespace = 1;
   string name = 2;
-  string client_id = 3;  // todo, not strictly needed
```

Comment:
Probably should mark this deprecated so we don't accidentally use index 3 in a future change

Reply:
yeah i can do that